fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/mv/IsNoCallChildrenValidator.java

```diff
@@ -15,25 +18,66 @@
 // If the children of scalarOperator are all ConstantOperator or ColumnRefOperator, will return true.
 // Currently, mainly support CaseWhenOperator.
 public class IsNoCallChildrenValidator extends ScalarOperatorVisitor<Boolean, Void> {
+    private final ColumnRefSet keyColumns;
+
+    private final ColumnRefSet aggregateColumns;
+
+    public IsNoCallChildrenValidator(ColumnRefSet keyColumns, ColumnRefSet aggregateColumns) {
+        this.keyColumns = keyColumns;
+        this.aggregateColumns = aggregateColumns;
+    }
+
+    @Override
+    public Boolean visitCastOperator(CastOperator operator, Void context) {
+        if (operator.getChild(0).isColumnRef()) {
+            return aggregateColumns.contains((ColumnRefOperator) operator.getChild(0));
+        }
+
+        if (operator.getChild(0).isConstantRef()) {
+            return ((ConstantOperator) operator.getChild(0)).isNull();
```

Comment:
Why Null is true,  other constant is false?

Reply:
SUM(NULL) alway NULL, SUM(1) on base table result is 10, but in mv maybe is 5 because mv will aggregate by key

Reply:
Comment it in code